geth/api/api.go

```diff
@@ -240,3 +268,46 @@ func (api *StatusAPI) NotifyUsers(message string, payload fcm.NotificationPayloa
 
 	return err
 }
+
+func (api *StatusAPI) ValidateJSONConfig(configJSON string) common.APIDetailedResponse {
+	var resp common.APIDetailedResponse
+	_, err := params.LoadNodeConfig(configJSON)
+
+	// Convert errors to common.APIDetailedResponse
+	switch err := err.(type) {
+	case validator.ValidationErrors:
+		resp = common.APIDetailedResponse{
+			Message:     "validation: validation failed",
+			FieldErrors: make([]common.APIFieldError, len(err)),
+		}
+
+		for i, ve := range err {
+			resp.FieldErrors[i] = common.APIFieldError{
+				Parameter: ve.Namespace(),
+				Errors: []common.APIError{
+					{
+						Message: fmt.Sprintf("field validation failed on the '%s' tag", ve.Tag()),
+					},
+				},
+			}
+		}
+	case error:
+		resp = common.APIDetailedResponse{
+			Message: fmt.Sprintf("validation: %s", err.Error()),
+		}
+	case nil:
+		resp = common.APIDetailedResponse{
+			Status: true,
+		}
+	}
+
+	return resp
+}
+
+func processError(err error) string {
```

Comment:
This is not needed because we don't want to print each error to stderr.

Reply:
Outputting to stderr was from existing code in lib/library.go, so removing would change the existing behaviour. Assuming from this comment this is ok?

Reply:
Yes, you can replace it with `log.Error("what happened", "error", err.Error())` like in other bindings. No unit tests are required to verify if that log actually got logged.